### DISCUSSION TITLE
Systray not following to active monitor wanted behaviour?

### DISCUSSION THREAD

**Discussion by mrwsl** [bakkeby/dwm-flexipatch#243] [Q&A] [answered]
Since I'm not sure if it is a bug: The systray isn't following to my second monitor. Is this wanted behaviour?
![2022-03-21_14-04](https://user-images.githubusercontent.com/8242787/159266806-f6743677-b637-40b8-af8a-6d870d674254.png)

**Comment by bakkeby** [comment, marked as answer]
It can be yes.

The first `0` here means that the systray will be shown on the first (primary) monitor. If you set this to `1` then the systray will only be shown on your secondary monitor.

https://github.com/bakkeby/dwm-flexipatch/blob/93f9d97e84bad4c5fae5d4381b96e77a1810ceaf/config.def.h#L499-L501

If you want the systray to follow the active monitor then you can change that first `0` to `'A'`.

https://github.com/bakkeby/dwm-flexipatch/blob/93f9d97e84bad4c5fae5d4381b96e77a1810ceaf/config.def.h#L473-L476

Note that setting this to `-1` would result in the systray flickering back and forth as the window can only be in one place at a time.

The same goes with the status depending on whether you want it to be static, shown on all monitors or only on the active monitor. Refer to places where the `statusmon` variable is used.

https://github.com/bakkeby/dwm-flexipatch/blob/93f9d97e84bad4c5fae5d4381b96e77a1810ceaf/config.def.h#L75-L82

**Reply by mrwsl** [reply]
I completely missed the comment in the `config.def.h` - sorry and thanks for explaining!
